src/Sarif.Multitool.Library/Rules/RuleId.cs

```diff
@@ -49,6 +49,13 @@ public static class RuleId
         public const string ProvideCheckoutPath = "GH1006";
         public const string ProvideFullyFormattedMessageStrings = "GH1007";
 
+        // Rules for GHAS and ADO Advanced Security
```

Comment:
>// Rules for GHAS and ADO Advanced Security [](http://example.com/codeflow?start=8&length=43)

another way is to separate in to a GHAS section and a ADO section, both order by id.